### PR TITLE
Return abort function

### LINKED ISSUE #11: Return a cleanup callback
```
let abort = loadScript();
abort();
```

should detach the element from the DOM, and prevent the callback from being fired (by ignoring it as a last resort).

### PR DESCRIPTION
**Pull request by bassettsj**
Fixes #11

### CODE REVIEW

**Comment by bassettsj** [comment]
There you go!

**Comment by defunctzombie** [comment]
dunno why I didn't get an email about this. Will take a look.

**Comment by defunctzombie** [comment]
Please update readme with new functionality :)

File: index.js [hunk 1]
```diff
@@ -24,6 +24,23 @@ module.exports = function load (src, opts, cb) {
     script.text = '' + opts.text
   }
 
+  setCallback(script, cb);
+  head.appendChild(script);
+
+  // Return an abort function
+  return function() {
+    setCallback(script, function() {})
+    script.parentElement.removeChild(script)
```
Comment (defunctzombie) [inline index.js]: does this tell the browser to stop the script from executing/finish loading?

File: test/index.js [hunk 2]
```diff
@@ -88,3 +88,10 @@ test('throw', function(done) {
   })
 });
 
+test('abort', function() {
```
Comment (defunctzombie) [inline test/index.js]: this test should use the `done` callback to wait a few moments to make sure that `test/hello.js` doesn't actually load (and your assertion is not called).

**Comment by vvo** [comment]
> dunno why I didn't get an email about this. Will take a look.

I stopped with github emails notifications and only use this: 
2015-09-09-105627_60x47_scrot

You can't miss anything on github with it. (like I have it blinking on zuul for day, sorry :))

**Comment by defunctzombie** [comment]
@rauchg thoughts on this?

**Pull request closed without merging**